Renderer/CKEditorRenderer.php

```diff
@@ -24,17 +25,41 @@
  */
 class CKEditorRenderer implements CKEditorRendererInterface
 {
+    /**
+     * @var JsonBuilder
+     */
+    private $jsonBuilder;
+    /**
+     * @var RouterInterface
+     */
+    private $router;
+    /**
+     * @var Packages
+     */
+    private $assetsPagages;
     /**
      * @var ContainerInterface
      */
     private $container;
+    /**
+     * @var Request
+     */
+    private $request;
 
     /**
+     * @param JsonBuilder $jsonBuilder
+     * @param RouterInterface $router
+     * @param Packages $packages
+     * @param RequestStack $requestStack
      * @param ContainerInterface $container
      */
-    public function __construct(ContainerInterface $container)
+    public function __construct(JsonBuilder $jsonBuilder, RouterInterface $router, Packages $packages, RequestStack $requestStack, ContainerInterface $container)
     {
+        $this->jsonBuilder = $jsonBuilder;
+        $this->router = $router;
+        $this->assetsPagages = $packages;
         $this->container = $container;
```

Comment:
Injecting the container is unnecessary I think. It it used to manually decide which templating engine to use, while if you are using Twig, it will automatically alias the `templating` service to `twig`. I think you can simply use the `Symfony\Component\Templating\EngineInterface` and inject the `templating` service instead of the container. Let the framework deal with deciding which engine to use.